### ISSUE TITLE
Install and run GITM on Ubuntu with Intel ifort

### ISSUE THREAD

**Issue by Prof-CW** [aaronjridley/GITM#18]
The steps provided for installing GITM do not work for Intel ifort. I have worked out how to get GITM to install and run, as described below for others who may have the Intel Fortran compiler.

Here is what I did:

1. git clone the GITM repository as described in the README

2. If you go ahead and do
   ./Config.pl -install -earth -compiler=ifort
then all the f90 source codes compile ok. The static libraries, obj and mod files all appear. However, the ifort link step errors with a whole bunch of undefined variables and there is no GITM.exe produced.

3. Here are the deatils of the software on my Dell laptop:
 OS:  Ubuntu 20.04.6 LTS

 gfortran --version
 GNU Fortran (Ubuntu 10.5.0-1ubuntu1~20.04) 10.5.0

 gcc --version
 gcc (Ubuntu 10.5.0-1ubuntu1~20.04) 10.5.0

 ifort --version
 ifort (IFORT) 2021.12.0 20240222

 mpiifort --version
 ifort (IFORT) 2021.12.0 20240222

4. The commands that worked for me were:
    ./Config.pl -install -earth -compiler=mpiifort
    make                    # check that GITM.exe and postProcess.exe appear in the /src dir after the make has finished
    make rundir
    cd run
    mpirun -np 4 ./GITM.exe
   cd UA
   ./pGITM
   cd data
etc...

In summary, using the -compiler=mpiifort (not ifort) worked for me.

**Comment by Adim-a11y** [comment]
Hi,
Thank you very much for this work. I just have a question.
Can it run on Windows?
